### PR TITLE
Update hostTaskSynopsis.h

### PR DESCRIPTION
**Pull request by mkinsner**
Editorial fix

### CODE REVIEW

**Review by TApplencourt — APPROVED** [review]

**Comment by TApplencourt** [comment]
I guess when we run `clang_format`, it was confused by the `...` and messed it up

**Review by gmlueck — APPROVED** [review]

File: adoc/headers/hostTask/hostTaskSynopsis.h [hunk 1]
```diff
@@ -41,12 +41,10 @@ class interop_handle {
 class handler {
   ...
```
Comment (tomdeakin) [inline adoc/headers/hostTask/hostTaskSynopsis.h:42]:
```suggestion
  // ...
```

File: adoc/headers/hostTask/hostTaskSynopsis.h [hunk 2]
```diff
@@ -41,12 +41,10 @@ class interop_handle {
...
-      public
-      :
+      public:
 
       template <typename T>
-      void
-      host_task(T&& hostTaskCallable);
+      void host_task(T&& hostTaskCallable);
 
   ...
```
Comment (tomdeakin) [inline adoc/headers/hostTask/hostTaskSynopsis.h:49]:
```suggestion
  // ...
```

**Comment by gmlueck** [comment]
@mkinsner: I noticed that there was a second code snippet that had exactly the same problem just a couple pages later in the spec.  I fixed this and also changed the indentation in 958d6ad19f61c39f01382b8c6c647dfc6ff743cd.  If you agree with that update, please give this comment a thumbs-up.  I'll wait for that before merging.

**Review by keryell — APPROVED** [review]
Thanks.

**Pull request merged**